pages/reference/hardware/versioning.md

```diff
@@ -12,40 +12,101 @@ In balenaCloud the device type is also used to define what architecture containe
 
 ### Designations
 
-When looking at all the supported device types in balenaCloud, you will notice that there are a number of different designations. Here we will discuss a few of these designations in more detail.
+Device types may have one of several designations, detailed below:
 
 #### Official
-All device types that are not designated with `COMMUNITY` or `PRIVATE` are considered officially supported boards.
+These are officially supported boards. If not discontinued, or not a private device type only accessible to customers paying for custom device type support.
 
 Official device types are:
 - Run through a full suite of testing on real hardware each time an OS version is released for the device type.
-- We endevour to ensure that all the hardware peripherals on the device type are functional and accessible. This includes things like i2c, SPI, audio, etc.
+- This test suite verified balenaOS specific funcitonality, including compatibililty with balena cloud and hostOS updates and rollbacks.
```

Comment:
This test suite verifies balenaOS specific functionality, including compatibility with balenaCloud and hostOS updates and rollbacks.